sw/device/silicon_creator/lib/drivers/spi_device.c

```diff
@@ -285,12 +285,14 @@ static_assert(kBfptTablePointer % sizeof(uint32_t) == 0,
 /**
  * BFPT 12th Word
  * --------------
- * [31: 9]: Erase/program suspend/resume (not supported, 0x0)
+ * [31:31]: Deep powerdown support (not supported, 0x1)
```

Comment:
Thanks for catching this! But I think deep powerdown is the MSb of the 14th DWORD, this should be "suspend/resume support"